code/game/objects/structures/safe.dm

```diff
@@ -157,6 +157,17 @@ FLOOR SAFES
 
 /obj/structure/safe/attackby(obj/item/I, mob/user, params)
 	if(open)
+		if(istype(I, /obj/item/disk/nuclear) && iscarbon(user))
+			var/mob/living/carbon/C = user
+			if(C.mind && C.mind.assigned_role == "Captain")

```

Comment:
I mean, I know this is a meme PR, but didn't @AnturK say that checking this sort of thing was bad practice? Or is that only with antagonists?

Reply:
This can probably be used to check if say a ling is actually a captain or not.